docs/resources/custom_location.md

```diff
@@ -0,0 +1,46 @@
+---
+page_title: "cpln_custom_location Resource - terraform-provider-cpln"
+subcategory: "Custom Location"
+description: |-
+---
+
+# cpln_custom_location (Resource)
+
+Manages an org's [Custom Location](https://docs.controlplane.com/reference/location#byok-locations).
+
+## Declaration
+
+### Required
+
+- **name** (String) Name of the Custom Location.
+- **tags** (Map of String) Key-value map of resource tags.
```

Comment:
Tags are not required, they are optional. We also need to add description as optional. Look at this example for reference.

https://github.com/controlplane-com/terraform-provider-cpln/blob/main/docs/resources/group.md?plain=1#L19